.github/.cspell.json

```diff
@@ -0,0 +1,74 @@
+{
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/vscode-extension/**",
+    "**/.git/**",
+    ".vscode",
+    "megalinter",
+    "package-lock.json",
+    "report"
+  ],
+  "language": "en",
+  "noConfigSearch": true,
+  "words": [
+    "AAVE",
+    "Autotask",
+    "Autotasks",
+    "Bignumber",
+    "CTOKEN",
+    "CTOKENADDRESS",
+    "Codespell",
+    "forta",
+    "IERC",
+    "Liquidatable",
+    "MULITSIG",
+    "Mult",
+    "NEWTEST",
+    "Offchain",
+    "REPORTERADDRESS",
+    "UNDERLYINGTOKENADDRESS",
+    "USDP",
+    "Unitroller",
+    "autotask",
+    "autotasks",
+    "bignumber",
+    "calldatas",
+    "cbtc",
+    "ceth",
+    "codespell",
+    "collateralized",
+    "creds",
+    "disctribution",
+    "forta",
```

Comment:
Nit: `forta` is present twice.  Should one of them have a capital letter at the beginning? e.g. `Forta`

Reply:
sorted, lowedcased, and deduped in commit b12701395a494e5ca3ef3dd51c7c14423fe82fa1